documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -0,0 +1,517 @@
+[[getting-started]]
+= Introduction to Hibernate Reactive
+
+:example: https://github.com/hibernate/hibernate-reactive/tree/master/example
+
+Before you start, we recommend taking a quick look at the example
+program in the {example}[`example`] directory, which shows off all
+the "bits" you'll need to get your own program up and running.
+
+== Setting up a reactive Hibernate project
+
+Setting up a project that uses Hibernate Reactive isn't hard at all.
+
+If you're using Hibernate Reactive outside of the Quarkus environment,
+you'll need to:
```

Comment:
Do we need to mention Quarkus here? Can't we just say something like:
``
You need these dependencies ... or, if you are using Quarkus, these extensions...
```

Reply:
Sure, if you have the list of which extensions are needed...

Reply:
For Hibernate Reactive is this one:
```
<dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-hibernate-reactive</artifactId>
        </dependency>
```
plus, the connector for the db:
```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-reactive-pg-client</artifactId>
        </dependency>
```

There is probably one for mysql and DB2 but I haven't checked

Reply:
ALso the deployment:
```
        <dependency>
            <groupId>io.quarkus</groupId>
            <artifactId>quarkus-hibernate-reactive-deployment</artifactId>
            <scope>provided</scope>
            <version>${quarkus-plugin.version}</version>
        </dependency>
```

Reply:
I just remembered that we could link to the `code.quarkus.io` for the generation of a project with reactive: https://code.quarkus.io/?g=org.acme&a=code-with-quarkus&v=1.0.0-SNAPSHOT&b=MAVEN&c=org.acme.ExampleResource&s=r1s.8XW.fmW.ih0&cn=code.quarkus.io

Reply:
Ahah, that looks good. Thanks.

Reply:
That's good enough for now, though we might still want to add a Quarkus/Panache chapter later.